plugin/wavesurfer.regions.js

```diff
@@ -269,19 +269,22 @@ WaveSurfer.Region = {
     bindInOut: function () {
         var my = this;
 
+        my.firedIn = false;
+        my.firedOut = false;
+
         var onPlay = function () {
-            my.firedIn = false;
-            my.firedOut = false;
```

Comment:
Would probably be cleaner with onPlay not there.  For those users who have custom code in onPlay, but are trying to stay current with the latest version of wavesurfer, it's easier for them if it's left there.
You want me to remove and re-submit the PR?